samples/angular/MusicStore/MusicStore.csproj

```diff
@@ -0,0 +1,45 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
```

Comment:
On all of your samples and template projects, consider adding `<IsPackable>false</IsPackable>`. This will allow you to execute `dotnet pack` on the solution and get packages for only those projects that should actually produce nuget packages.